packages/client/src/components/formConfig/formTools/components.tsx

```diff
@@ -129,5 +129,16 @@ export const RegisterFormFieldIds = ({
     .flatMap((section) => formSectionToFieldIdentifiers(defaultForm, section))
     .map((identifier) => getFieldId(event, identifier, defaultForm))
 
-  return children(fieldIds)
+  const filteredFieldIds = fieldIds.filter((fieldId) => {
+    if (
+      !fieldId.includes('seperator') &&
+      !fieldId.includes('informantType') &&
+      !fieldId.includes('contactPoint')
+    ) {
+      return true
+    }
+    return false
+  })
+
+  return children(filteredFieldIds)
```

Comment:
@sadmananik Could we use this [function](https://github.com/opencrvs/opencrvs-core/blob/b6978b48acc9c3144e790f70bb773d624d8fb148/packages/client/src/components/formConfig/SectionNavigation.tsx#L55) instead of filtering `informantType` and `contactPoint`?